source/getting-started/build-host-setup.rst

```diff
@@ -13,17 +21,26 @@ Install the following software:
 * NASM

```

Comment:
One Dx user mentioned that its better to specify what NASM and iASL mean at a very high level like a one liner so that it is clear on what it actually is.